server/api/product.js

```diff
@@ -1,21 +1,184 @@
+//Agrega un nuevo producto ejemplo 
+//
+//{
+//		"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6InBlcGUxIiwiaWF0IjoxNDY4MDE1NDM4fQ.CFKDtiulzTTAGj09-xw8HMUxdAtGvnpL8oJmKmaau_o",
+//	    "name":"correa",
+//	    "description":"correa para ropes copada",
+//	    "price":50030,
+//	    "type":"product"
+//}
+//
+//Devuelve el mismo objeto con el product_id agregado: 
+//
+///{
+//		"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6InBlcGUxIiwiaWF0IjoxNDY4MDE1NDM4fQ.CFKDtiulzTTAGj09-xw8HMUxdAtGvnpL8oJmKmaau_o",
+//	    "name":"correa",
+//	    "description":"correa para ropes copada",
+//	    "price":50030,
+//	    "type":"product"
+//	    "product_id": 10
+//}/
 export function post(request, response, next){
+	//TODO: Como no puedo obtener el token desde react, no se lo pido mas
+	next.ifError(request.hasSessionError());
 	next.ifError(request.params.validationError({
-		// Tiene que devolver un id del producto
-		// TODO: validar token
-		// TODO: agregar fotos
-		required: ['token', 'name', 'description', 'price','type'],
+		required: ['name', 'description', 'price', 'type', 'images'],
 		properties: {
-			token: {type: 'string'},
-			name: {type: 'string', minLength: 5},
-			description: {type: 'string', minLength: 10},
-			price: {type: 'number'},
+			name: {type: 'string', minLength: 4},
+			description: {type: 'string', minLength: 2},
+			price: {type: 'number', minimum: 0},
 			type: {
-				"enum" : [ "producto" , "servicio" , "servicio-profesional"]
+				"enum" : [ "product" , "service" , "professional-service"]
+			},
+			images: {
+				type: 'array',
+				minLength: 1
```

Comment:
Le habia pifeado es [`minItems`](http://epoberezkin.github.io/ajv/keywords.html#keywords-for-arrays)